src/provider/mlkzg.rs

```diff
@@ -53,29 +53,17 @@ where
   E::Fr: TranscriptReprTrait<E::G1>,
   E::G1Affine: TranscriptReprTrait<E::G1>, // TODO: this bound on DlogGroup is really unusable!
 {
-  fn compute_challenge(
-    C: &E::G1Affine,
-    y: &E::Fr,
-    com: &[E::G1Affine],
-    transcript: &mut impl TranscriptEngineTrait<NE>,
-  ) -> E::Fr {
-    transcript.absorb(b"C", C);
-    transcript.absorb(b"y", y);
-    transcript.absorb(b"c", &com.to_vec().as_slice());
```

Comment:
This `com` does need to be part of the transcript

Reply:
Yes, agree. Thanks. It is being computed inside the proving / verifying so it should improve entropy of the transcript.